src/app/(admin)/settings/components/sections/profile.tsx

```
@@ -0,0 +1,40 @@
+    return (
+        <div>
+            <h2 className="text-xl font-semibold mb-4 text-gray-900 dark:text-white">Profile Settings</h2>
+            <div className="space-y-6">
+                <div className="flex items-center space-x-4">
+                    <div className="h-16 w-16 bg-gray-200 dark:bg-gray-700 rounded-full"></div>
+                    <button className="px-4 py-2 text-sm font-medium text-gray-700 dark:text-gray-200 border border-gray-300 dark:border-gray-600 rounded-md hover:bg-gray-50 dark:hover:bg-gray-700">
+                        Change Photo
+                    </button>
+                </div>
+                <div className="space-y-4">
+                    <div>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)